src/components/Tabs/__tests__/TabItem.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {
+  expect,
+  test,
+} from '@playwright/experimental-ct-react';
+import {
+  propTests,
+  mixPropTests,
+} from '../../../../tests/playwright';
+import { TabItemForTest } from './TabItem.story';
+import { isActivePropTest } from './_propTests/tabItem/isActivePropTest';
+
+test.describe('Tabs', () => {
+  test.describe('visual', () => {
+    [
+      ...propTests.defaultComponentPropTest,
+      ...propTests.afterLabelPropTest,
+      ...mixPropTests([
+        propTests.beforeLabelPropTest,
+        isActivePropTest,
+      ]),
+    ].forEach(({
+      name,
+      onBeforeTest,
+      onBeforeSnapshot,
+      props,
+    }) => {
+      test(name, async ({
+        mount,
+        page,
+      }) => {
+        if (onBeforeTest) {
+          await onBeforeTest(page);
+        }
+        const component = await mount(
+          <TabItemForTest {...props} />,
+        );
+        if (onBeforeSnapshot) {
+          await onBeforeSnapshot(page, component);
+        }
+        const screenshot = await component.screenshot();
+        expect(screenshot).toMatchSnapshot();
+      });
+    });
+  });
+
+  test.describe('non-visual', () => {
+    test('id', async ({ mount }) => {
+      const id = 'test-id';
+      const label = 'label';
+
+      const component = await mount(
+        <TabItemForTest
+          id={id}
+          label={label}
+        />,
+      );
+
+      expect(component.locator(`li[id=${id}]`)).toBeDefined();
+      await expect(component.getByRole('link')).toHaveAttribute('id', `${id}__link`);
+      await expect(component.getByText(label)).toHaveAttribute('id', `${id}__label`);
+    });
+  });
+
+  test.describe('functionality', () => {
+    test('call on click callback', async ({ mount }) => {
```

Comment:
`onClick` is optional, but I am glad you test it! Neverhless, if it is possible, test also `href` functionality which is default behavior. See `TextLink.spec.tsx` where everything is tested. Also, keyboard accessibility should be part of testing as well.